mscore/pagesettings.h

```diff
@@ -22,29 +22,42 @@
 #define __PAGESETTINGS_H__
 
 #include "ui_pagesettings.h"
+#include "libmscore/mscore.h"
+#include "libmscore/page.h"
 
 namespace Ms {
 
 class MasterScore;
 class Score;
 class Navigator;
 
+// For converting from Ms::Units to a two-char string used as a suffix
+static const char* unitSuffixes[] = {"px", "mm", "in", "sp"};
```

Comment:
just search for setSuffix. Inyout arrax use something like ` QT_TRANSLATE_NOOP3("suffixUnits", "mm", "millimeter unit")` and in the cose rather than the litetal `"mm"` use `qApp()->translate("suffixUnits", suffixUnits[1])`, mabe instead of the digit and an enum, for better readability, `int(SuffixUnit::MM)`.